lib/node_modules/@stdlib/math/base/special/factorial2/lib/factorial2.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+// MODULES //
+
+var exp2 = require( '@stdlib/math/base/special/exp2' );
+var gamma = require( '@stdlib/math/base/special/gamma' );
+var isnan = require( '@stdlib/math/base/utils/is-nan' );
+var isInteger = require( '@stdlib/math/base/utils/is-integer' );
+var isNegativeZero = require( '@stdlib/math/base/utils/is-negative-zero' );
+var sqrt = require( '@stdlib/math/base/special/sqrt' );
+var DOUBLE_FACTORIALS = require( './doublefactorials.json' );
```

Comment:
This should be the last `require` statement. Order of `requires`:

* Builtins; e.g., `path`, `Buffer`, etc.
* External packages; e.g., `@stdlib/math/base/special/sqrt`.
* Relative modules; e.g., `./doublefactorials.json`.